src/disk_interface.cc

```diff
@@ -71,7 +71,7 @@ TimeStamp TimeStampFromFileTime(const FILETIME& filetime) {
 
 TimeStamp StatSingleFile(const string& path, string* err) {
```

Comment:
Now that you're always converting to a `std::wstring`, this should be a `StringPiece` instead of a `std::string`

Reply:
@jlonnberg